### ISSUE TITLE
RuntimeException "Not logged in" - login failure on version 3.33.0

### ISSUE THREAD

**Issue by chrisolof** [platformsh/platformsh-cli#717]
I just installed platformsh-cli 3.33.0 on Ubuntu 18.04 and cannot log in:

```bash
$ platform --verbose login

                      
  [RuntimeException]  
  Not logged in       
                      

Exception trace:
 () at phar:///home/chris/.platformsh/bin/platform/vendor/platformsh/client/src/Connection/Connector.php:214
 Platformsh\Client\Connection\Connector->getOauth2Plugin() at phar:///home/chris/.platformsh/bin/platform/vendor/platformsh/client/src/Connection/Connector.php:312
 Platformsh\Client\Connection\Connector->getClient() at phar:///home/chris/.platformsh/bin/platform/src/Service/Api.php:214
 Platformsh\Cli\Service\Api->getClient() at phar:///home/chris/.platformsh/bin/platform/src/Command/Auth/BrowserLoginCommand.php:57
 Platformsh\Cli\Command\Auth\BrowserLoginCommand->execute() at phar:///home/chris/.platformsh/bin/platform/vendor/symfony/console/Command/Command.php:252
 Symfony\Component\Console\Command\Command->run() at phar:///home/chris/.platformsh/bin/platform/vendor/symfony/console/Application.php:964
 Symfony\Component\Console\Application->doRunCommand() at phar:///home/chris/.platformsh/bin/platform/src/Application.php:266
 Platformsh\Cli\Application->doRunCommand() at phar:///home/chris/.platformsh/bin/platform/vendor/symfony/console/Application.php:248
 Symfony\Component\Console\Application->doRun() at phar:///home/chris/.platformsh/bin/platform/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at phar:///home/chris/.platformsh/bin/platform/bin/platform:19
 require() at /home/chris/.platformsh/bin/platform:10
```

I'm getting the same results when I try ```platform --verbose auth:password-login```, though it does let me supply my username and password before failing the same way.  Any ideas?

**Comment by chrisolof** [comment]
In case it helps - I'm running PHP 7.2.5.

**Comment by chrisolof** [comment]
Just tried moving down to PHP 7.1 and no dice - same error.

**Comment by pjcdawkins** [comment]
No ideas yet, could you run:

```
PLATFORMSH_CLI_DEBUG=1 platform login
```

but please do not post the full result here, at least not without removing out all the lines containing `Authorization`.

**Comment by pjcdawkins** [comment]
Ah I do have an idea, it looks like a regression in v3.33.0 :(...

**Comment by pjcdawkins** [comment]
Many thanks for the report: this is fixed in version 3.33.1 which will be available via `platform up` in an hour (allowing for caching), although you can update faster via:

```sh
platform up --manifest https://raw.githubusercontent.com/platformsh/platformsh-cli/master/dist/manifest.json
```

**Comment by chrisolof** [comment]
Fixed!  Thank you!!
